docs/_snippets/storybook-auto-docs-mdx-docs-imports.md

```diff
@@ -1,8 +1,6 @@
-```mdx renderer="common" language="mdx"
-{/* Checkbox.mdx */}
-
+```mdx filename="Checkbox.mdx" renderer="common" language="mdx"
+{/* TK: vet this against recommendation and filter out if not supported by Svelte's CSF*/}
```

Comment:
it's supported, but the import needs to be `import * as CheckboxStories from './Checkbox.stories.svelte';` in Svelte CSF